### PR TITLE
ITSASU-1833 - Added the getting prepared section, simplified structur…

### PR DESCRIPTION
**Pull request by AlexRimmerHMRC**
…e, fixed small screen view

### CODE REVIEW

File: test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala [hunk 1]
```diff
@@ -58,59 +58,81 @@ class SignUpConfirmationViewSpec extends ViewSpec {
...
 
-          "contains the online HMRC services section" which {
-            def onlineHmrcServices: Element = testMainContent.selectNth(".govuk-grid-column-one-half", 2)
+          if(yearIsNext) {
+            "contains the online HMRC services section first" which {
+              def onlineHmrcServices: Element = testMainContent.selectNth(".col", 1)
 
-            "contains a heading" in {
-              onlineHmrcServices.selectHead("h3").text() mustBe SignUpConfirmationMessages.section2onlineServicesHeading
-            }
```
Comment (sb-dev) [inline test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala:66]: The following test got deleted for when `yearIsNext` is `true`:
```
            "contains a heading" in {
              onlineHmrcServices.selectHead("h3").text() mustBe SignUpConfirmationMessages.section2onlineServicesHeading
            }
```

File: test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala [hunk 2]
```diff
@@ -58,59 +58,81 @@ class SignUpConfirmationViewSpec extends ViewSpec {
             testMainContent.selectNth("h2", 2).text() mustBe SignUpConfirmationMessages.section2heading
           }
 
-          "contains the online HMRC services section" which {
-            def onlineHmrcServices: Element = testMainContent.selectNth(".govuk-grid-column-one-half", 2)
+          if(yearIsNext) {
+            "contains the online HMRC services section first" which {
```
Comment (sb-dev) [inline test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala:62]: Minor changes:
```suggestion
            "contains the online HMRC services section in first position" which {
```

File: test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala [hunk 3]
```diff
@@ -58,59 +58,81 @@ class SignUpConfirmationViewSpec extends ViewSpec {
...
-              "contains next year paragraph 2" in {
+              "contains a paragraph 2" in {
                 onlineHmrcServices.selectNth("p", 2).text() mustBe SignUpConfirmationMessages.section2onlineServicesNextYearParagraph2
               }
-            } else {
-              "contains this year paragraph" in {
-                onlineHmrcServices.selectHead("p").text() mustBe SignUpConfirmationMessages.section2onlineServicesThisYearParagraph
+
+            }
+            "contains the getting prepared section second" which {
```
Comment (sb-dev) [inline test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala:79]:
```suggestion
            "contains the getting prepared section in second position" which {
```

File: test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala (hunk 1 above)
Comment (AlexRimmerHMRC) [reply test/views/individual/incometax/subscription/SignUpConfirmationViewSpec.scala:66]: Hmm, thought I put that in, will add now

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -33,14 +33,25 @@ body:not(.js-enabled) {
...
-  width: 100%;
 }
 
-.col {
-  flex: 1;
+.boxout:first-child {
+  margin-left: 0;
+}
+
+@media (min-width: 40.0625em) {
```
Comment (aug24) [inline app/assets/stylesheets/application.scss:47]: Ooh - what problem did this solve?

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -33,14 +33,25 @@ body:not(.js-enabled) {
...
+    .boxout {
+      margin-left: govuk-spacing(2);
```
Comment (aug24) [inline app/assets/stylesheets/application.scss:49]: This first-child and other children doesn't quite do what we had before, does it?
It won't make two equal sized boxes.  The left one will be bigger.

File: app/assets/stylesheets/application.scss (hunk 4 above)
Comment (AlexRimmerHMRC) [reply app/assets/stylesheets/application.scss:47]: The boxes shouldn't be side by side when displayed on a smaller screen

**Review by aug24 — APPROVED** [review]

**Pull request merged**